stable/velero/values.yaml

```diff
@@ -19,16 +19,14 @@ podAnnotations: {}
 # Resource requests/limits to specify for the Velero deployment. Optional.
 resources: {}
 
-# Pod priority class name to use for the Velero deployment. Optional.
-priorityClassName: {}
-
-# Init containers to add to the Velero deployment's pod spec. Optional.
+# Init containers to add to the Velero deployment's pod spec. At least one plugin provider image is required.
```

Comment:
this would be very helpful, otherwise i wont know where to add the plugins to the chart